pkg/oc/oc.go

```diff
@@ -94,10 +93,8 @@ func DeleteResource(resourceType, name string) {
 }
 
 func CheckProjectExists(projectName string) bool{
-	if strings.Contains(cmd.Run("oc", "project", projectName).String(), "error:"){
-		fmt.Println("******************************")
+	if strings.Contains(cmd.Run("oc", "project", projectName).String(), "error: A project named \""+projectName+"\" does not exist"){
```

Comment:
This will not work for non-admin users, maybe just check that output contains `error` without any message :shrug: